communications.md

```diff
@@ -2,95 +2,62 @@
-- Optionally snooze items that need future attention. Archive them as soon as they're done, or no longer needed.
-- Archive the rest.
-
-Opt-out: Use filters to avoid receiving emails sent to a shared email address that you are not interested in.
 
 #### - Project Management & Product Issue Documentation: Asana -
 
-Opt-in: Join projects that you are interested in or obligated to.
-
-Minimize Distractions: Similar to Slack, instead of striving to achieve "Notification Zero", try to be as close as possible to "Everything Zero". Defer any information that is not work-related
-
-Documentation - Transparency: Provide as much detail as possible for other team members for context and opt-in to a task or issue if necessary.
-
-Opt-out: Leave projects that you are no longer interested in or obligated to.
-
-## Second line of Communication: Meeting
+Our Asana workspace is commonly used for project management e.g task allocation, task state (in progress or completed), and any other process we deem it fit to be used for (e.g Office Availability, calender scheduling, etc.)
+Opt-in to join projects/tasks that you are interested in or obligated to.
 
-If a meeting is necessary, budget the amount of time needed appropriately. Each meeting is the collective productivity value of all its participants. Consider each participant to be on average contributing $200/hr; it is always good to judge the need/value of each meeting against the economic value of the meeting (what other productive tasks could these participants be doing?).
 
-#### - Meetings & Events Tool: Google Calendar -
+#### Meetings
+If a meeting is necessary, we have a meeting space that can be pre-booked and used appropriately. Please be mindful of the duration time as we presently have limited private space for meetings and others might be waiting in line.
 
-Opt-in: Subscribe to calendars you are interested in or obligated to.
+###### Events Tool: Google Calendar -
+Opt-in to subscribe to calendars you are interested in or obligated to.
+You can set up meetings by using event invitations and book one of the available rooms in the google invitation to avoid disappointment.
 
-Set up meetings: Use event invitations to set up a meeting and book one of the available rooms in the google invitation to avoid disappointment.
+###### Preparing for & During Meetings: 
+To ensure that meetings are productive and are respectful of everyone's time and productivity, feel free to try out the steps below:
 
-Preparing for & During Meetings: To ensure that meetings are productive and are respectful of everyone's time and productivity, you may do the following:
-
-1. Aggressively timebox them. If a meeting was set to be an hour-long, ask if it can be shortened to 30 minutes; if it was set to be 30 minutes long, ask if it can be shortened to 15 minutes... better yet, see if you can do it without a meeting.
+1. Timebox meeting to ensure the duration is convenient for every or most participants.
```

Comment:
removing the materialized example of collective time-spending makes this just "talk", and also harder for people to realize the true damage of abusing meetings.

Reply:
Yes that is exactly how it should be, "TALK". Guidelines are not to be stressed out so using terms like "Aggressively" is somewhat inappropriate in this context. 

**Key**: 
**Guidelines** are not to be stressed or over emphasized. **Warnings** can have all that.

I honestly do not see recursively asking if a meeting can be shortened (1hr -> 30min ->15min -> No) to be productive but rather evasive. That is why I took that bit out.
I think in scheduling meetings, we suggested that the meeting host be considerate of the time of the other parties, so if they have 1 hour to spare then it is fine if not it can be discussed as availability time is different for individuals.

Reply:
No. Guidelines should not just be empty talks.

Reply:
Consider how 4-hour block Is more productive than 4x 1 hour segmented time slots.

To shorten and then even more on the timeframe, and ideally translate that more effective written means, is the core of this suggestion.